creator-node/package.json

```diff
@@ -20,6 +20,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@audius/libs": "1.2.93",
+    "@solana/web3.js": "^1.31.0",
```

Comment:
Should we pin this version?

Reply:
oops, meant to pin but only changed the version. I'll pin it soon